src/backend/joanie/settings.py

```diff
@@ -373,6 +373,36 @@ class Base(Configuration):
         },
     }
 
+    # Signature
+    JOANIE_SIGNATURE_BACKEND = {
+        "backend": values.Value(
+            "joanie.signature.backends.lex_persona.client.LexPersonaClient",
+            environ_name="JOANIE_SIGNATURE_BACKEND",
+            environ_prefix=None,
+        ),
+        "configuration": {
+            "base_url": values.Value(
+                "", environ_name="JOANIE_LEX_PERSONA_BASE_URL", environ_prefix=None
+            ),
+            "consent_page": values.Value(
+                "",
+                environ_name="JOANIE_LEX_PERSONA_CONSENT_PAGE",
+                environ_prefix=None,
+            ),
+            "session_user": values.Value(
+                "", environ_name="JOANIE_LEX_PERSONA_USER", environ_prefix=None
+            ),
+            "signature_page": values.Value(
+                "",
+                environ_name="JOANIE_LEX_PERSONA_SIGNATURE_PAGE",
+                environ_prefix=None,
+            ),
+            "token": values.Value(
+                "", environ_name="JOANIE_LEX_PERSONA_TOKEN", environ_prefix=None
+            ),
```

Comment:
default values should probably be None instead of empty string so you can gracefully handle the cases of missing options for a given backend?

Reply:
I can agree with you but...I've followed the `JOANIE_PAYMENT_BACKEND` example though.
```
    JOANIE_PAYMENT_BACKEND = {
        "backend": values.Value(
            "joanie.payment.backends.payplug.PayplugBackend",
            environ_name="JOANIE_PAYMENT_BACKEND",
            environ_prefix=None,
        ),
        "configuration": {
            "secret_key": values.Value(
                "", environ_name="PAYPLUG_SECRET_KEY", environ_prefix=None
            ),
            "public_key": values.Value(
                "", environ_name="PAYPLUG_PUBLIC_KEY", environ_prefix=None
            ),
        },
    }
```

Let me know what suits you best. I can apply `None` as default values in order to gracefully handle the cases of missing info for a given backend. I have no issues in both ways.